domain_path.module

```diff
@@ -31,7 +30,7 @@ function domain_path_field_info_alter(&$info) {
  * Implements hook_module_implements_alter().
  */
 function domain_path_module_implements_alter(&$implementations, $hook) {
```

Comment:
I know `hook_field_widget_info_alter` is how pathauto does it, but this still seems like a bit of a messy way of extending the classes (we're already running into trouble with two modules trying to override the same class) it seems like we should be decorating the FieldTypePluginManager and WidgetPluginManager instead. But as this works for now will leave it as is and possibly look at it later.